SimpleRabbit.NetCore/Service/ParallelMessageHandler.cs

```diff
@@ -80,14 +82,14 @@ public bool Process(BasicMessage message)
         }
 
         /// <summary>
-        ///     Must be provided to decompose the message to a TValue e.g. perform any deserialisation or object creation.
+        /// Must be provided to decompose the message to a TValue e.g. perform any deserialisation or object creation.
         /// </summary>
         /// <param name="message"></param>
         /// <returns>The decomposed message</returns>
         protected abstract TValue Get(BasicMessage message);
```

Comment:
Implementation of this might fail and consumer should be able to specify the type of `Acknowledgement` in that case. Same with for `GetKey` abstract method

Reply:
Do we want to just return a bool representing success or failure?